nipype/interfaces/base.py

```diff
@@ -715,7 +715,7 @@ def _get_filecopy_info(self):
 
 
 class BaseInterfaceInputSpec(TraitedSpec):
-    ignore_exception = Bool(help="Print an error message instead \
+    ignore_exception = Bool(default_value=False, help="Print an error message instead \
```

Comment:
We probably want to make ignore_exception a BaseInterfaceSpec class attribute, this could be a good point to make this deprecation.

Reply:
ok, i added it to my todo. want to finish other things first